nixos/modules/services/misc/ollama.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }: let
+
+  cfg = config.services.ollama;
+
+in {
+
+  options = {
+    services.ollama = {
+      enable = lib.mkEnableOption (
+        lib.mdDoc "Server for local large language models"
+      );
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+
+    systemd = {
+      services.ollama = {
+        wantedBy = [ "multi-user.target" ];
+        description = "Server for local large language models";
+        after = [ "network.target" ];
+        environment = {
+          HOME = "/var/lib/ollama";
+          OLLAMA_MODELS = "/var/lib/ollama/models";
```

Comment:
Please use systemd specifiers for these directories: https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Specifiers